release_notes/44-release-notes.adoc

```diff
@@ -594,6 +595,20 @@ This releases updates the following items to patch vulnerabilities:
 * Go has been updated to release 1.20.12.
 * The `golang.org/x/net` module has been updated from release v0.22.0 to v0.23.0.
 
+[id="resolved-in-version-443_{context}"]
+=== Resolved in version 4.4.3
+
+*Release date*: 3 June 2024
+
+This release provides the following bug fixes:
+
+* Fixed an issue with `Updated Secured Cluster Identity` duplicate events in telemetry.
+* Fixed an issue where Scanner V4 was unable to scan images from registries that have untrusted TLS certificates, even when TLS verification was set to `skipped`.
+
+This release contains the following updates:
+
+* Updated Collector version to include fixes for kernels 6.7 and RHEL 9.4.
```

Comment:
Do we need to provide more info on "6.7"? (I'm not sure what it is.)

Reply:
@kcarmichael08  Yep, it turns out we better give more information here, as it might be necessary to motivate upgrading from 4.4.0-4.4.2 to this patch release. I'll prepare something shortly, is that ok?

Reply:
Since we've decided to take a path of least resistance, not much extra information is needed. I would propose following:

```
Updated Collector version to support RHEL 9.4 and and distributions, which use Linux kernels newer than v6.7-rc5.
```

Is it a bit more clear here what is 6.7?

Reply:
Yes, thank you!